package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "restaurant",
+  "version": "1.0.0",
+  "description": "This project is a simple restaurant website built with HTML, CSS, and Js. It displays the \r ## Built With",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
Let us improve our build scripts by replacing our build script with this =>  `"build": "webpack --mode production"` and also add this => ``"dev": "webpack --mode development"``